lib/reporting/identity_verification_report.rb

```diff
@@ -354,7 +358,14 @@ def query
       QUERY
     end
 
+    class FakeCloudwatchClient
+      def fetch(**)
+        []
+      end
+    end
+
     def cloudwatch_client
+      return FakeCloudwatchClient.new # FIXME so, so much
```

Comment:
Obviously this cannot stay here. Doing the obvious thing in ReportMailerPreview didn't avoid us hitting the real CloudWatch client, so I threw this in here just to get a preview working. Need to dig more.